core/block.js

```diff
@@ -1264,6 +1329,9 @@ Blockly.Block.prototype.jsonInit = function(json) {
   if (json['output'] !== undefined) {
     this.setOutput(true, json['output']);
   }
+  if (json['maxInstances'] !== undefined) {
```

Comment:
This shouldn't be part of the block definition as it's a per workspace setting, not a per block setting.